### PR TITLE
IL-196: Personal account

### PR DESCRIPTION
**Pull request by Sunagatov**
…ser package

### CODE REVIEW

File: src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java [hunk 1]
```diff
@@ -30,4 +31,13 @@ public UserDto getUserById(final UUID userId) throws UserNotFoundException {
                     return new UserNotFoundException(userId);
                 });
     }
+
+    @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.READ_COMMITTED, readOnly = true)
```
Comment (yevr19) [inline src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java:35]: just readonly would be ok here

File: src/main/resources/api-specs/user-openapi.yaml [hunk 2]
```diff
@@ -17,12 +17,12 @@ tags:
...
 
 paths:
-  /api/v1/users/{userId}:
+  /api/v1/users/:
     get:
       tags:
         - "User"
       summary: "Get an user by its ID"
-      description: "Get a detailed user profile for the given user ID."
+      description: "Retrieve a detailed user profile for the given user ID."
```
Comment (yevr19) [inline src/main/resources/api-specs/user-openapi.yaml:25]: why did you remove this id from path in url? The method name and desc are kinda confusing

Comment (Sunagatov) [reply src/main/resources/api-specs/user-openapi.yaml:26]: Fixed

File: src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java (hunk 1 above)
Comment (Sunagatov) [reply src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java]: Fixed

File: src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java
Comment (nnick44) [inline src/main/java/com/zufar/icedlatte/user/api/SingleUserProvider.java:1]: unused import line 10,11

File: src/main/java/com/zufar/icedlatte/user/exception/InvalidOldPasswordException.java
Comment (nnick44) [inline src/main/java/com/zufar/icedlatte/user/exception/InvalidOldPasswordException.java:1]: unused import line 5
